### PR TITLE
Automated tagging and releases

### PR DESCRIPTION
**Pull request by petricm**
BEGINRELEASENOTES

NEW: Extend the CI to produce tags, releases and builds based on branch name

ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
Hum, why is my commit in there as well ?

**Comment by petricm** [comment]
I removed you commit

File: docs/70_release.md [hunk 1]
```diff
@@ -1,45 +1,38 @@
...
+git clone git@github.com:DIRACGrid/DIRACOS.git
+cd DIRACOS
 ```
 
-## Updating the release.notes
-
-This has to be done using [this script](https://github.com/DIRACGrid/DIRAC/blob/integration/docs/Tools/GetReleaseNotes.py) in the DIRAC repository:
-
-Documentation on how to use it can be found [here](https://dirac.readthedocs.io/en/latest/DeveloperGuide/DevelopmentModel/ReleaseProcedure/index.html?highlight=GetReleaseNotes#release-notes)
+No get all the PRs since the last tag. This has to be done using [this script](https://github.com/DIRACGrid/DIRACOS/blob/master/docs/Tools/GetReleaseNotes.py) in the DIRACOS repository by executing
```
Comment (chaen) [inline docs/70_release.md]: Now ?

**Comment by chaen** [comment]
Also, do we need to give permissions somehow to diracos user or not needed ?

**Pull request merged**